common/Perf/Azure.Sample.Perf/Batch/MockReceiver.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Azure.Sample.Perf.Batch
+{
+    public class MockReceiver
+    {
+        private readonly Random _random = new Random();
```

Comment:
Each instance of `MockReceiverTest` creates its own `MockReceiver`, so there should be no concurrent access to a single `Random` instance.  But even if there were it would be good enough -- the randomness is just to roughly simulate receiving a random number of messages each call.